### ISSUE TITLE
Doesn't build in windows correctly

### ISSUE THREAD

**Issue by themortalcoil** [yahoo/ember-intl#159]
The tests do not pass and an app using 2.0 doesn't build in windows correctly.  I believe this is because of the use of Glob which has a dependency on Minimatch which can't correctly parse Windows folders.
https://github.com/isaacs/minimatch/issues/64

**Comment by jasonmit** [comment]
I don't have access to a windows machine, any possibility you'll be able to triage this or find a volunteer?

**Comment by jasonmit** [comment]
> The tests do not pass

Can you provide more details

**Comment by themortalcoil** [comment]
Yeah, the error is in TranslationBlender.updateCache()  The use of glob there returns an empty array and causes the following error log to be triggered. 
> [ember-intl] en default locale missing from 'translations' folder". 

The files exist and so do the cached copies.  Our fix was to do the following :
``` javascript
    var defaultTranslationPath = path.join(includePath, this.options.defaultLocale) + '\.yaml';
```
to basically hardcode the value. We lose glob's ability to match on different extensions.

And as for the tests they fail with the same error we were getting during build

**Comment by jasonmit** [comment]
Okay, I setup a VM.  Will dig into this tonight.

**Comment by jasonmit** [comment]
Discovered the issue was around glob's `nocase` option.  I've decided to just remove the option since the PR to fix glob has not yet been mergedd: https://github.com/isaacs/node-glob/pull/146

**Comment by jasonmit** [comment]
Please validate #master to ensure your issue is resolved.  I will then version a new beta.

**Comment by themortalcoil** [comment]
Thank you, I'll test ASAP

**Comment by jasonmit** [comment]
Verified it on my VM and published 2.0.0-beta.9

**Comment by themortalcoil** [comment]
Works for me. Thanks again
